ext/opcache/jit/vtune/jitprofiling.c

```diff
@@ -62,7 +62,7 @@
 #include <windows.h>
 #pragma optimize("", off)
 #endif /* ITT_PLATFORM==ITT_PLATFORM_WIN */
-#if ITT_PLATFORM != ITT_PLATFORM_MAC && ITT_PLATFORM != ITT_PLATFORM_FREEBSD
+#if ITT_PLATFORM != ITT_PLATFORM_MAC && ITT_PLATFORM != ITT_PLATFORM_FREEBSD && ITT_PLATFORM != ITT_PLATFORM_OPENBSD && ITT_PLATFORM != ITT_PLATFORM_NETBSD
 #include <malloc.h>
```

Comment:
Just tried simplify dropping this include and it seems to work. Don't see anything in the code either that would need malloc.h. I'd prefer to drop this over adding more platform detection.

Reply:
All right I did not test on Linux, I ll change it then.